src/reminder/reminder.interface.ts

```diff
@@ -0,0 +1,40 @@
+export interface Recurring {
+    recurringDateOfMonth: number
+    recurringDay: number
+}
+
+export interface Reminder {
+    rid: number
+    startingDateTime: Date
+    title: string
+    note: string
+    isRemindCaretaker: boolean
+    importanceLevel: string
+    imageid: string
+    isDone: boolean
+}
```

Comment:
Since this is not a DTO, why do you need to explicitly declare this instead of using the `Reminder` entity class?